content/en/community/contributing/code/releasing/pyxform.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Releasing Pyxform"
+linkTitle: "Releasing Pyxform"
+weight: 5
+description: >
+    Instructions for updating medic/pyxform to new XLSForm/pyxform releases
+relatedContent: >
+    releases/
+aliases: >
+  /contribute/code/releasing/pyxform
+---
+
+The `medic/pyxform` repository is a fork of [XLSForm/pyxform](https://github.com/XLSForm/pyxform) with custom modifications for the CHT. This page documents the process for updating `medic/pyxform` to a new upstream release from XLSForm/pyxform, ensuring the custom changes are preserved and the process is replicable.
+
+## Overview
+
+- The `medic/master` branch contains the latest tagged code for pyxform-medic, with all custom commits applied on top of the target release from `xlsform/pyxform`.
+- When targeting a new release from `xlsform/pyxform`, create a new branch (e.g., `4.0.0-rebase`) off of `medic/master` and rebase it on top of the target tag.
+- Open a PR from the rebase branch for review.
+- Once approved, force-push the rebase branch to `medic/master` (do not merge the PR).
+- Tag the new release.
+
+## Prerequisites
+
+- Ensure all custom features are finalized and working as expected.
+- Have access to push to your fork of `medic/pyxform` and (for maintainers) to `medic/pyxform`.
```

Comment:
Let's drop this section.  We will have more to say below about testing the features and the repo access can be assumed...

```suggestion
```